src/ResourceManager/IotHub/Commands.IotHub/IotHub/AddAzureRmIotHubCertificate.cs

```diff
@@ -0,0 +1,119 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
If any of your cmdlets use long running SDK calls, please add the -AsJob parameter.  You can check if an operation is long running in the .NET SDK (any function that calls a "begin" function is long running).  In order to add the parameter, simply add to the parameter list:
````
[Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
public SwitchParameter AsJob { get; set; }
````
After that, you should be able to test the cmdlet with the parameter and see that a job is returned.  To add this to testing, simply use:
````
$job = Remove-AzureRmIotHubCertificate
$job | Wait-Job
$delete = $job | Receive-Job
````

Reply:
Yes, added this parameter